### ISSUE TITLE
Automatic tracking has stopped working with any source/tracker/manga/re-install

### ISSUE THREAD

**Issue by cronjobsasha** [tachiyomiorg/tachiyomi#5965]
### Steps to reproduce

1.  Track with either MAL on Anilist
2. Make sure "Update progress on Reading" is selected
3.  Add Berserk/Solo Levelling/My Perfect Girlfriend or any manga to the library
4. Make sure it's tracked
5. Read any new chapter 

### Expected behavior

- Expected chapter update automatically in MAL/Anilist
- Instead have to manually open the tracker from the app and update it.

### Actual behavior

* No update on MAL or Anilist - have to manually use the tracking chapter to update it.

### Tachiyomi version

0.12.2

### Android version

11

### Device

Samsung Ultra 21

### Other details

- All this used to work and not sure why it stopped working
- Tried to isolate problem with various things
- Switched from Anilist to MAL
- Re-installed app
- Re-install extensions
- Migrated manga from one tracker to another
- Cleared cookies
- Tried various random manga
- No matter what I do it never gets updated automatically
- Updating manually via app works so this is not a connection issue and I verified update chapter progress is selected.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by paulo27ms** [comment]
When did it stop working? Did it work at all in 0.12.2? Just for those series or everything? You can try downgrading to 0.12.1 to see if it starts working again.

**Comment by cronjobsasha** [comment]
it was not working in the earlier version. then i uninstalled it and
upgraded it to the latest version thinking but the issue still persists.

**Comment by arkon** [comment]
Are you actually reading chapters to the last page?

**Comment by cronjobsasha** [comment]
Yes, I read multiple chapters across multiple manga with multiple sources
and multiple trackers to be sure. Manual app chapter update seems to work.
This seems to suggest something is going wrong in between the recording it
as we read across chapters.

**Comment by paulo27ms** [comment]
Give examples for some chapter lists you're reading.

**Comment by cronjobsasha** [comment]
Solo Levelling (read last two,)
One Piece (read last two)
Berserk (read from beginning)

All sourced from Mangajar extension.

I have uninstalled and reinstalled the apps and extension and cleared data
and cookies and behavior persists.

**Comment by arkon** [comment]
All the chapters are considered as chapter 0:

https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/src/en/mangajar/src/eu/kanade/tachiyomi/extension/en/mangajar/MangaJar.kt#L125

This is a source issue.
